packages/shared/src/utils/contract/contract.js

```diff
@@ -59,6 +59,8 @@ export const getDigitInfo = (digits_info, contract_info) => {
     };
 };
 
+export const getTotalProfit = contract_info => contract_info.bid_price - contract_info.buy_price;
```

Comment:
Are `bid_price` and `buy_price` always present on contract_info?

Reply:
yeah, it will be always present